packages/packages/commonwell-sdk/src/common/util.ts

```diff
@@ -6,8 +6,8 @@ export function getId(object: Person): string {
   return url.substring(url.lastIndexOf("/") + 1);
 }
 
-export function getIdPatient(object: Patient): string {
+export function getIdTrailingSlash(object: Patient): string {
```

Comment:
Was using it for more than just patient urls

Reply:
But this always expects a patient?

Reply:
Realized that i actually didnt need it for another type so i switched it back